site/en/docs/handbook/how-to/add-an-api/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Add an API
+description: 'Add an API to the reference area.'
+date: 2020-12-17
+---
+
+## Add an API
+
+To add a reference page for a Chrome API, create a folder and `index.md` file inside
+`site/en/docs/extensions/reference/`. The folder name doesn't directly relate to the API
+itself—control which API is rendered via frontmatter like this:
+
+```md
+---
+api: browserAction
+---
+
+Something about this API. This text is completely optional. If you don't want to write anything,
+then you should include a comment like <!-- This is intentionally empty -->.
+```
+
+The API name is assumed to be under "chrome.apiName". If you are documenting an API like
+"chrome.input.ime", then by convention, the folder name should be "input_ime" but the API in your
+frontmatter shold still be `api: input.ime`.
+
+The existence of this page will cause the API itself to be displayed in the
+[top-level reference page][1]. If it's a Chrome Apps-only API, it will appear at the very bottom
+of that page, rather than in Stable, Beta or Dev.
+
+## API Source
+
+The API reference is generated via the published APIs in [chrome-types on NPM][2]. It's possible
```

Comment:
```suggestion
The API reference is generated via the published APIs in [chrome-types on npm][2]. It's possible
```

I think the npm folks lowercase it
![image](https://user-images.githubusercontent.com/1066253/104247485-c3756400-541c-11eb-8bf0-94bb104eced5.png)